assets/js/interactivity/directives.js

```diff
@@ -1,4 +1,10 @@
-import { useContext, useMemo, useEffect, useLayoutEffect } from 'preact/hooks';
```

Comment:
I tested the Product Button and Product Gallery blocks and can confirm they're working as expected with the changes added to this file.